core/web_admin.py

```
@@ -1219,38 +1524,119 @@ async def get_login_status(self) -> dict[str, Any]:
+            logger.debug(f"[WebAdmin] disconnect before clear session failed: {exc}")
+
+        await TelegramClientWrapper.disconnect_and_clear_cache(session_path)
+
+        for path in self._session_files(session_path):
+            if not os.path.exists(path):
+                continue
+            os.makedirs(backup_dir, exist_ok=True)
+            try:
+                shutil.copyfile(path, os.path.join(backup_dir, os.path.basename(path)))
+                backed_up = True
+            except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)